src/common/syncjournalfilerecord.h

```diff
@@ -55,6 +55,8 @@ class OCSYNC_EXPORT SyncJournalFileRecord
     bool isDirectory() const { return _type == ItemTypeDirectory; }
     bool isFile() const { return _type == ItemTypeFile || _type == ItemTypeVirtualFileDehydration; }
     bool isVirtualFile() const { return _type == ItemTypeVirtualFile || _type == ItemTypeVirtualFileDownload; }
+    QString pathFromUtf8() const { return QString::fromUtf8(_path); }
```

Comment:
For a "fromSomething" I'd expect the "Something" as parameter while this takes no parameter. What about "pathAsString()"? I'm even wondering if just "path()" would be enough... since we were toying with the idea of switching it all to QString instead, having such a `QString path() const` would be a nice API to slowly port against and when _path isn't used directly anymore it's a sign it could be switched to QString. Sounds like a potentially nice strategy for such a changer over a longer period of time.